crates/tauri-plugin-holochain-service-consumer/guest-js/holochain-env/index.ts

```diff
@@ -48,19 +49,35 @@ function injectHolochainClientEnv(installedAppId: string, port: number, token: U
 (window as any).injectHolochainClientEnv = injectHolochainClientEnv;
 
 // Define function to install app, get app websocket, and inject magic config variables
-async function setupApp(installedAppId: string, source: number[], networkSeed: string) {
+async function setupApp(installedAppId: string, source: number[], networkSeed: string, enableApp: boolean) {
   if (window.location.origin !== 'http://tauri.localhost') return;
 
+  await (window as any).__TAURI_INTERNALS__.invoke('plugin:holochain-service-consumer|connect');
+
   // Check if happ is installed
   const { installed } = await (window as any).__TAURI_INTERNALS__.invoke('plugin:holochain-service-consumer|is_app_installed', { installedAppId });
   
   // Install happ if not already
   if(!installed) {
     await (window as any).__TAURI_INTERNALS__.invoke('plugin:holochain-service-consumer|install_app', { installedAppId, source, roleSettings: {}, networkSeed });
+
+    // Enable App
+    if(enableApp) {
```

Comment:
Enable app after installing, if specified in the plugin initialization function (some apps will not want to do this).